git/Cheatsheat.md

```diff
@@ -0,0 +1,98 @@
+# Git Cheatsheet
+
+This file contains guides such as **how to write good commit message**, **useful git commands** and their explanation.
+
+## Table of Contents
+
+- [Writing good commit message](https://github.com/lambdaworkshub/contributing-guide/blob/master/git/Cheatsheat.md#writing-good-commit-message)
+- [Useful Git commands and how to use them](https://github.com/lambdaworkshub/contributing-guide/blob/master/git/Cheatsheat.md#useful-git-commands-and-how-to-use-them)
+- [How to rebase properly?](https://github.com/lambdaworkshub/contributing-guide/blob/master/git/Cheatsheat.md#how-to-rebase-properly)
+
+### Writing Good Commit Message
+
+Provided [Erlang's guide](https://github.com/erlang/otp/wiki/writing-good-commit-messages) gives rules for writing good commit messages.
+
+- Write your commands in **imperative**.
+
+    **GOOD:** `Remove unused code`
+
+    **WRONG:** ~~`Removed unused code`~~
+
+- Don't put **punctuation marks** at the end of commit message.
+
+    **GOOD:** `Remove unused code`
+
+    **WRONG:** ~~`Remove unused code.`~~
+
+- Use short commit messages - up to 50 characters.
+
+    **GOOD:** `Remove unused code`
+
+    **WRONG:** ~~`In this commit we've removed unused code along with classes`~~
+
+- Don't put more statements in one commit message. Use description instead. Description should be wrapped to 80 characters maximum.
+
+    **GOOD:**
+    ```
+    Remove unused code
+
+    Removed 'getAllUsers' function from 'UserDAO' class.
+    Performed cleanup on 'Users' controller. Removed unused
+    config file.
+    ```
+
+    **WRONG:** ~~`Remove 'getAllUsers' function. Cleanup 'Users' controller. Remove unused config.`~~
+
+- Leave second line blank when writing descriptions.
+
+    **GOOD:**
+    ```
+    Remove unused code
+
+    Removed 'getAllUsers' function from 'UserDAO' class.
+    Performed cleanup on 'Users' controller. Removed unused
+    config file.
+    ```
+
+    **WRONG:**
+    ```
+    Remove unused code
+    Removed 'getAllUsers' function from 'UserDAO' class.
+    Performed cleanup on 'Users' controller. Removed unused
+    config file.
+    ```
+
+### Useful Git commands and how to use them
+
+We assume that you've already known basic Git commands. So, we won't list you basic commands. We'll focus on specific commands that could be useful sometimes.
+
+1. `git commit --amend`
+    This command is used when we don't want to make two commits with same purpose. For example, if we push some code and we notice that we forget to delete `print` from our code, then there's no sense to make another commit with `Remove unused 'print' command`. Then, we just remove it with use of `--amend` flag instead.
```

Comment:
Use something like _This command is used to avoid creation of two commits with the same or similar purpose._